src/Layouts/Partials/AlignItemsPanel.vala

```diff
@@ -19,48 +19,47 @@
  * Authored by: Giacomo "giacomoalbe" Alberini <giacomoalbe@gmail.com>
  */
 public class Akira.Layouts.Partials.AlignItemsPanel : Gtk.Grid {
-    public weak Akira.Window window { get; construct; }
+    public unowned Lib.ViewCanvas view_canvas { get; construct; }
 
     public int current_button_column { get; set; default = 0; }
```

Comment:
We can make this `private` since it's only used inside this class.

Reply:
This is not possible due to the fact that the assignment is done using the Object function which does require the construct keyword in the property assignment and properties must be public. I'll leave it public for now, tell me if there is another method to deal with those cases.